### PR TITLE
adding eclipse-bluechi

### PR DESCRIPTION
**Pull request by odra**
Adding an cleaned up eclipse-bluechi folder to run the blueprint.

### CODE REVIEW

File: eclipse-bluechi/README.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# Eclipse BlueChi
+
+![Smart trailer blueprint](../docs/diagrams/bluechi.png)
```
Comment (ladatz) [inline eclipse-bluechi/README.md:3]: Depending on in which order this PR and #19 get merged, this path may need to be updated

File: eclipse-bluechi/README.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+This document describe how to run the Eclipe SDV Orchestrator blueprint using Eclipse BlueChi.
```
Comment (ladatz) [inline eclipse-bluechi/README.md:5]:
```suggestion
This document describes how to run the Eclipse Software Orchestration blueprint using Eclipse BlueChi.
```

File: eclipse-bluechi/README.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+This section describes how to run this blueprint using Eclipse Bluechi.
+
+### Devcontainers + VSCode
+
+This section describes how to run the blueprint using the VSCode devcontainer extension.
+
+#### Prerequisites
+
+* Ensure that the [Remote Development extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) installed in VSCode
```
Comment (ladatz) [inline eclipse-bluechi/README.md:26]:
```suggestion
* Ensure that the [Remote Development extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) is installed in VSCode
```

File: eclipse-bluechi/README.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+          app: freyja
+      template:
+        metadata:
+          labels:
+            app: freyja
+        spec:
+          hostNetwork: true
+          containers:
+            - name: local
+              image: sdvblueprint.azurecr.io/sdvblueprint/eclipse-freyja/local-with-ibeji:0.1.0
```
Comment (ladatz) [inline eclipse-bluechi/README.md:156]: Consider updating this to the updated path for this image, since the one listed here is private.

File: eclipse-bluechi/README.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+This is valid for any systemd defined service, simply run `journalctl -xeu {SERVICE_NAME}`
+
+#### Podman
+
+You can also list all active containers by running `podman ps` and then `podman logs {CONTAINER_NAME_OR_ID}` to
+get logs from the container using podman. Replace `{CONTAINER_NAME_OR_ID}` with the container's name or ID.
+
+## Running the Smart Trailer Example with BlueChi's devcontainer
+Inside of the [devcontainer](#two-development-environments):
+1. Follow the instructions in [Starting All the Services](#starting-all-the-services) to start up the in-vehicle stack.
```
Comment (ladatz) [inline eclipse-bluechi/README.md:198]: It doesn't look like these two links lead to anywhere
"#two-development-environments" and "#starting-all-the-services"

File: eclipse-bluechi/README.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+1. Run the script `start_trailer_applications_bluechi.sh` to monitor for the trailer to be connected. It can be found at `in-vehicle-stack/scenarios/smart_trailer_use_case/scripts/start_trailer_applications_bluechi.sh`.
```
Comment (ladatz) [inline eclipse-bluechi/README.md:199]: All of the smart trailer application files will be contributed soon in another PR. This documentation may need to change based on the new paths

**Review by ladatz — COMMENTED** [review]
Please consider my comments

File: eclipse-bluechi/README.md (hunk 6 above)
Comment (odra) [reply eclipse-bluechi/README.md:199]: whoops, I missed it. I was going to remove this part as well.

File: eclipse-bluechi/README.md (hunk 5 above)
Comment (odra) [reply eclipse-bluechi/README.md:198]: ah, just left overs from the previous doc. Let me remove them.

File: eclipse-bluechi/README.md (hunk 1 above)
Comment (eriksven) [reply eclipse-bluechi/README.md:3]: @ladatz @odra do you have a preference of the order in which I should then merge #18 and #19? Considering the comment by @ladatz, I guess that this link needs to adapted anyway, once #19 gets merged.

File: eclipse-bluechi/README.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+You can now enter into the devcontainer and iteract with Eclipse BlueChi:
+
+```sh
+docker exec -it $container_id_or_name /bin/bash
+bluechictl list-units
+```
+
+### Docker/Podman
+
+You can also directly run this blueprint directly with Docker or Podman.
```
Comment (eriksven) [inline eclipse-bluechi/README.md:74]: maybe consider removing one of the two "directly"

File: eclipse-bluechi/README.md (hunk 4 above)
Comment (eriksven) [reply eclipse-bluechi/README.md:156]: for reference, the workflow is part of #17

**Comment by eriksven** [comment]
Could you please add Copyright headers specifying the used license, especially to the newly added files? At the moment, the used license is Apache 2.0. See the [Eclipse Project Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) and #16 for more details.

File: eclipse-bluechi/README.md (hunk 1 above)
Comment (odra) [reply eclipse-bluechi/README.md:3]: You can merge #19, I will rebase my pr.

**Comment by odra** [comment]
@eriksven would that be needed even for readme files?

**Comment by eriksven** [comment]
@odra thank you for adding the headers and it should be ok to not have them in the Readme files. Merging #19 caused a merge conflict and as discussed earlier it moved some of the referenced images. Can you have quick look to fix this? Other than that, I think we are good to merge this PR.

**Comment by odra** [comment]
@eriksven just rebased it.
